modules/claims_api/spec/lib/claims_api/bd_spec.rb

```diff
@@ -151,20 +155,22 @@
     end
 
     describe '#build_body' do
+      let(:tracked_items) { ews.tracked_items }
+
       it 'builds an L705 (5103) body correctly' do
-        result = subject.send(:build_body, doc_type: 'L705', file_name: '5103.pdf', claim_id: ews.claim_id)
+        result = subject.send(:build_body, doc_type: 'L705', file_name: '5103.pdf', claim_id: ews.id, tracked_items:)
 
-        expected = { data: { systemName: 'VA.gov', docType: 'L705', claimId: '60897890',
-                             fileName: '5103.pdf', trackedItemIds: [] } }
+        expected = { data: { systemName: 'VA.gov', docType: 'L705', claimId: '43fc03ab-86df-4386-977b-4e5b87f0817f',
```

Comment:
Here and below: why are we changing the expected `claim_id` from `ews.claim_id` to `ews.id`?